batch/python/tiles_geojson.py

```diff
@@ -0,0 +1,96 @@
+import json
+from concurrent.futures import ProcessPoolExecutor, as_completed
+from typing import Any, Dict, List, Tuple
+
+from geojson import Feature, FeatureCollection
+from pyproj import CRS, Transformer
+from shapely.geometry import Polygon
+from shapely.ops import unary_union
+
+from errors import GDALError
+from gdal_utils import run_gdal_subcommand
+
+
+def to_4326(crs: CRS, x: float, y: float) -> Tuple[float, float]:
+    transformer = Transformer.from_crs(
+        crs, CRS.from_epsg(4326), always_xy=True
+    )
+    return transformer.transform(x, y)
+
+
+def extract_metadata_from_gdalinfo(gdalinfo_json: Dict[str, Any]) -> Dict[str, Any]:
+    """Extract necessary metadata from the gdalinfo JSON output."""
+    corner_coordinates = gdalinfo_json["cornerCoordinates"]
+
+    crs: CRS = CRS.from_string(gdalinfo_json["coordinateSystem"]["wkt"])
+    metadata = {
+        # NOTE: pixetl seems to always write features in tiles.geojson in
+        # epsg:4326 coordinates (even when the tiles themselves are
+        # epsg:3857). Reproduce that behavior for compatibility. If that
+        # ever changes, remove the call to to_4326 here.
+        "extent": [
+            *to_4326(crs, *corner_coordinates["lowerLeft"]),
+            *to_4326(crs, *corner_coordinates["upperRight"]),
+        ],
+        "name": gdalinfo_json["description"],
```

Comment:
It looks like the tiles.geojson that we create with pixetl have a whole bunch more properties in them (looks like most that are printed by gdalinfo), but I'm assuming you've verified that we only need the 'extent' and 'name' properties - since those are the only ones you are generating beside the Polygon coordinates?

Reply:
Huh. I actually had those other attributes in there, but took them out because (I swear) I looked at a tiles.geojson that didn't have them and I was trying to reproduce it. I must have been mistaken, I guess? Maybe I was accidentally looking at a custom geojson I had lying around? I don't know, but you're right, I see all that stuff in recent tiles.geojsons. I'll put it back.